scripts/apps/search/components/fields/takekey.jsx

```diff
@@ -4,6 +4,10 @@ import PropTypes from 'prop-types';
 export function takekey(props) {
     const {gettextCatalog} = props.svc;
 
+    if (props.item.rewrite_of) {
+        return React.createElement('span', {className: 'takekey', key: 'takekey'},
+            gettextCatalog.getString('duplicated'));
+    }
     if (props.item.anpa_take_key) {
         return React.createElement('span', {className: 'takekey', key: 'takekey'},
             gettextCatalog.getString(props.item.anpa_take_key));
```

Comment:
What does this function return if none of the conditions are met? Is it ok that it doesn't render anything? At least we should add a `return null`.